Haroun/Code/Data Extraction/eICU/Production/COHORT_BASELINE_P1.sql

```diff
@@ -0,0 +1,125 @@
+with allICU as 
+  (select 
+    uniquepid, 
+    patienthealthsystemstayid, 
+    patientunitstayid,	
+    gender,
+    age,
+    ethnicity,
+    hospitaladmitoffset,
+    unitdischargeoffset,
+    hospitaldischargeoffset,
+    unitdischargestatus,
+    hospitaldischargestatus,
+    ROW_NUMBER() OVER (PARTITION BY patienthealthsystemstayid ORDER BY hospitaladmitoffset) as ICU_rank
+  --   hospitalid,
+  --   wardid,
+  --   apacheadmissiondx,
+  --   admissionheight,
+  --   hospitaladmitsource,
+  --   hospitaldischargeyear,
+  --   hospitaldischargetime24, 
+  --   hospitaladmittime24,
+  --   hospitaldischargelocation,
+  --   unittype,
+  --   unitadmitsource,
+  --   unitvisitnumber,
+  --   unitstaytype,
+  --   admissionweight,
+  --   dischargeweight,
+  --   unitdischargetime24,
+  --   unitdischargelocation		
+  from `physionet-data.eicu_crd.patient` -- table of ICU stays confusingly called "patients"
+  where unitDischargeOffset > 720 -- los is longer than 12 hours
+  ), 
+
+firstICU as (
+  select * from allICU 
+  where ICU_rank = 1 -- first icu stay within the same hospital admission
+  ),
+
+on_mech_vent as(
+  select icu.*, ox.vent_start, ox.vent_end,	ox.oxygen_therapy_type,	ox.supp_oxygen	
+  from firstICU icu 
+  inner join `NMB_eICU.eicu_oxygen_therapy` ox
+  on icu.patientunitstayid = ox.icustay_id
+  where ox.ventnum = 1),
```

Comment:
`NMB_eICU.eicu_oxygen_therapy` also includes records of non-mechanical ventilation which you are currently not excluding. Is that an issue?

Reply:
That's a good point! Thank you for pointing out. I changed it to only invasive (oxygen_therapy_type = 4). Cohort size decreased by 20%